text-unidecode is an optional dependency; we have unidecode
in our tree which is picked up instead.

Index: setup.py
--- setup.py.orig
+++ setup.py
@@ -12,7 +12,7 @@ package = 'slugify'
 python_requires = ">=3.7"
 here = os.path.abspath(os.path.dirname(__file__))
 
-install_requires = ['text-unidecode>=1.3']
+install_requires = []
 extras_requires = {'unidecode': ['Unidecode>=1.1.1']}
 test_requires = []
 
